Cargo.toml

```diff
@@ -50,7 +57,11 @@ fxhash = "0.2.1"
 
 [features]
 # Serialization with serde 1.0
-serde-1 = ["serde"]
+serde-1 = ["serde", "atone/serde", "griddle/serde"]
+rayon = ["rayon_", "atone/rayon", "griddle/rayon"]
```

Comment:
I believe this means enabling either `serde-1` or `rayon` will always force `atone` and `griddle` to be built, regardless of `amortize`.

I don't think we really need all those transitive features anyway, just `atone/rayon`. Our `serde` support only needs plain iteration, and for `rayon` we only touch the vector.

Reply:
I think this is why CI failed on 1.36 thumb for `test-nostd`, because that enables `indexmap/serde-1`.

Reply:
Ah, yes, looks like you're right: https://github.com/rust-lang/cargo/issues/3494. And to quote from that:
> I don't believe it's possible to express what you're thinking here today with Cargo features.

But your point that we may not need those features is a good one — I'll play around with that. I think even for the vector, we _may_ be able to get away with not having the rayon feature on `atone` either.